DataLink.tex

```diff
@@ -1076,7 +1076,7 @@ \subsection{Example: Self-Describing Service}
             value="http://example.com/mylinks" />
      <GROUP name="inputParams">
        <PARAM name="ID" datatype="char" arraysize="*"
-              value="" ref="primaryID"/>
+              value="" />
```

Comment:
Since this is a self-describing service descriptor (a datalink service describing itself, the lack of ref seems more correct (the doc would not have an element with ID="primaryID". So this is ok.

Reply:
This fix  is for an Erratum on DataLink 1.0. It will disappear in 1.1 because the example is changed.